### ISSUE TITLE
Zoom in (CTRL + ) doesn't works

### ISSUE THREAD

**Issue by soleroks** [smartfrigde/armcord#564]
**Describe the bug**
Zoom out works as expected but not zoom in. I'm stuck with a Discord only readable by ants.

**To Reproduce**
Spam CTRL -

**Expected behavior**
Client should zoom in when pressed CTRL +

**Comment by smartfrigde** [comment]
What keyboard layout do you use?

**Comment by soleroks** [comment]
I don't think it has nothing with keyboard layout since zoom out works but anyway, Turkish Q.
![TurkishQWERTY](https://github.com/ArmCord/ArmCord/assets/82513212/edf431d2-ce60-438a-8939-be6c3a85ecc5)

**Comment by ritiek** [comment]
@soleroks as a workaround, i think hitting the ALT key should show the top menu bar in armcord which should have the zoom option.
